<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitForSoftimage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="banner_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 bold italic 18pt &quot;Trebuchet MS&quot;;
color: rgb(220, 220, 220);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 20, 20, 255), stop:1 rgba(40, 40, 40, 255));</string>
     </property>
     <property name="text">
      <string>GitForSoftimage </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="contents_layout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="branch_layout">
       <item>
        <widget class="QComboBox" name="branch_comboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="branch_button">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Manage</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="separator_line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Commit Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="history_groupBox">
       <property name="title">
        <string>History</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="history_tableWidget">
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="ok_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>branch_comboBox</tabstop>
  <tabstop>branch_button</tabstop>
  <tabstop>message_lineEdit</tabstop>
  <tabstop>history_groupBox</tabstop>
  <tabstop>history_tableWidget</tabstop>
  <tabstop>ok_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>ok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>history_tableWidget</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>history_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branch_comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>branch_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branch_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>branch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>history_groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>history_toggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok_clicked()</slot>
  <slot>history_clicked()</slot>
  <slot>branch_changed()</slot>
  <slot>branch_clicked()</slot>
  <slot>history_toggled()</slot>
 </slots>
</ui>
